types/airtable/index.d.ts

```diff
@@ -1,13 +1,23 @@
-// Type definitions for airtable 0.5
+// Type definitions for airtable 0.8
 // Project: https://github.com/airtable/airtable.js
 // Definitions by: Brandon Valosek <https://github.com/bvalosek>
 //                 Max Chehab <https://github.com/maxchehab>
+//                 Sam Garson <https://github.com/samtgarson>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
+// TypeScript Version: 3.7
 
-export class Airtable {
-    constructor(options?: Airtable.AirtableOptions);
-    base(appId: string): Airtable.Base;
+export type baseCreateFunctor = (tableName: string) => Table;
```

Comment:
Does this type need to be exported?

In general, I think we might be exporting more things than need exporting. What do you think?